storybook/src/docs/components/AppIcons.tsx

```diff
@@ -0,0 +1,78 @@
+import { Image, Paragraph } from '@amsterdam/design-system-react'
+
+const FigureGridStyle = { display: 'flex', alignItems: 'center', gap: '1em' }
```

Comment:
Alternatively we have a file for some storybook css: storybook/config/preview-body.html

Reply:
Check, I was not aware. Is some minor JS CSS in the Story okay though?

Reply:
Adjusted.